### PR TITLE
Add current timeslot and send message

### PR DESCRIPTION
**Pull request by Danzibob**
For front page message box

### CODE REVIEW

File: selector.go [hunk 1]
```diff
@@ -1,7 +1,25 @@
 package myradio
 
 import (
-	"github.com/UniversityRadioYork/myradio-go/api"
+//"github.com/UniversityRadioYork/myradio-go/api"
+)
+
+const (
```
Comment (Danzibob) [inline selector.go]: This was basically copied from the PHP-side implementation. Thought about having separate methods to check these properties but that would've been even more messy. Other suggestions welcome.

File: selector.go [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package myradio
+
+const (
+	// Values for the current selection/where it was selected from
+	SelectorStudio1 = 1
+	SelectorStudio2 = 2
```
Comment (Danzibob) [inline selector.go]: Copied from PHP implementation, but would be open to suggestions on how to make this less awful

Comment (Danzibob) [reply selector.go]: Also Codacy hates these comments, but I think an individual comment for each one of these isn't helpful to anyone

Comment (LordAro) [reply selector.go]: if they were sequential, you could've used `iota`, ...but they're not. I see no disadvantage to explicitly numbering them

**Pull request merged**